### PR TITLE
Fixed #111: Improved code quality

### LINKED ISSUE #111: Improve code quality
Referring to the files in _includes folder

### PR DESCRIPTION
**Pull request by samyakbvs**
- Removed spaces
- Indented code
- Clubbed like tags together

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
     <script src="https://use.fontawesome.com/releases/v5.0.0/js/all.js"></script>
+    <script src="javascripts/navigationbar.js"></script>
 
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width initial-scale=1" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="{{ site.description }}">
 
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
-    <meta name="description" content="{{ site.description }}">
```
Comment (anshumanv) [inline _includes/head.html:10]: Why removed this?

**Comment by samyakbvs** [comment]
@anshumanv I added it above again with other <meta> tags

**Review by anshumanv — APPROVED** [review]
LGTM

**Comment by samyakbvs** [comment]
Can you merge?

**Comment by samyakbvs** [comment]
@anshumanv

**Comment by anshumanv** [comment]
> Can you merge?

I can merge, but I need to wait for other reviewers. In open source the more reviews the better, so please stop asking for merge unless you have gained atleast 2 or more reviews.

**Comment by samyakbvs** [comment]
Okay I didn't know that sorry @anshumanv

**Review by akshatnitd — APPROVED** [review]
LGTM!

**Pull request merged**